iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/FileUploadTests.java

```diff
@@ -304,7 +304,8 @@ private void verifyNotification(FileUploadNotification fileUploadNotification, F
         try (InputStream inputStream = u.openStream())
         {
             byte[] testBuf = new byte[(int)fileUploadState.fileLength];
-            int testLen = inputStream.read(testBuf,  0, (int)fileUploadState.fileLength);
+            inputStream.read(testBuf,  0, (int)fileUploadState.fileLength);
+            int testLen = (int)fileUploadState.fileLength;
```

Comment:
The commit message explains this further, but Android SDKs return -1 when inputStream.read(...) reads no bytes. JDKs return 0 though. For consistency between JDK and android SDK, we shouldn't use this return value

Reply:
that is unfortunate ... good thinking.